app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -52,4 +54,15 @@ public function store(ReportbackRequest $request)
 
         return $this->item($reportback, $code);
     }
+
+    /**
+     * Update reportbackitem(s) that already exists.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function updateReportbackItems(Request $request)
+    {
+        $this->reportbackService->updateReportbackItems($request->all());
```

Comment:
Thanks, @sbsmith86! Added that logic in but still getting the same error message. It's also strange because when I do `dd(gettype($items)` it says it is an array (which is what it is looking for)?